pkg/controllers/jobset_controller.go

```diff
@@ -328,13 +324,55 @@ func (r *JobSetReconciler) createHeadlessSvcIfNotExist(ctx context.Context, js *
 	return nil
 }
 
+func (r *JobSetReconciler) executeSuccessPolicy(ctx context.Context, js *jobset.JobSet, ownedJobs *childJobs) error {
+	switch js.Spec.SuccessPolicy.Operator {
+	case jobset.OperatorAll:
+		return r.successPolicyAll(ctx, js, ownedJobs)
+	case jobset.OperatorAny:
+		return r.successPolicyAny(ctx, js, ownedJobs)
+	default:
+		return nil
```

Comment:
in the previous iteration I returned an error here, but in the comments we decided to move this validation to the api CEL Enum validation (All; Any) on the API itself, which I did (see [comments](https://github.com/kubernetes-sigs/jobset/pull/124#discussion_r1191735567)). Do we want both the validation in the API and the error returned here as well (perhaps it could alert us to a bug in the CEL validation)?

Reply:
combined successPolicy execution functions per https://github.com/kubernetes-sigs/jobset/pull/124#discussion_r1198308062 so this is no longer applicable